apps/docs/pages/guides/auth/phone-login/twilio.mdx

```diff
@@ -347,8 +347,30 @@ let { data, error } = await supabase.auth.verifyOtp({
 })
 ```
 
+## Twilio Verify
+
+Twilio Verify is an alternative service by Twilio which can replace Twilio Programmable Messaging (previous described). Developers may migrate from Twilio Messaging to:
+
+1. Make use of built in fraud detection
+2. Utilize Message Services instead of a registered phone number.
+
+Twilio Verify is currently [set up as a Phone Provider](/docs/img/guides/auth-twilio/twilio-verify.png)
+
+To set up Twilio Verify, you will need to:
+
+1. [Create a Verification Service on Twilio](https://support.twilio.com/hc/en-us/articles/360033309133-Getting-Started-with-Twilio-Verify-V2)
+2. Switch Phone Provider to Twilio Verify
+3. Fill in the Message Service ID field on the Dashboard with the ID of the Verification Service created in 1.
+
+Unlike other providers, the OTP expiry duration, message content fields, and rate limits are not configurable via the Supabase dashboard. Please head to Twilio Verify to configure these settings.
```

Comment:
Is this true? GoTrue rate limits don't apply for Twilio Verify?

Reply:
They do, but if Twilio Verify has tighter limits than GoTrue then Twilio Verify takes precedence. I'll update to refelcet that.